### PR TITLE
upgrade kubefirst marketplace pin to v2.4.17

### PR DESCRIPTION
**Pull request by johndietz**
existing Marketplace application

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [x] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [x] Tested that the application works with the proposed updates applied (including a screenshot).
* [x] Updated the version number of the app if that has changed.

### CODE REVIEW

**Comment by johndietz** [comment]
CleanShot 2024-08-13 at 23 04 30@2x

File: kubefirst/install.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 helm repo add kubefirst https://charts.kubefirst.com
 helm repo update
 
-helm install kubefirst --set=console.installMethod=civo-marketplace --set=global.cloudProvider=civo --set=global.clusterType=bootstrap --set=global.installMethod=civo-marketplace --set=console.ingress.enabled=false  --create-namespace --namespace kubefirst --version 2.4.13 --wait kubefirst/kubefirst
\ No newline at end of file
+helm install kubefirst --set=console.installMethod=civo-marketplace --set=global.cloudProvider=civo --set=global.clusterType=bootstrap --set=global.installMethod=civo-marketplace --set=console.ingress.enabled=false  --create-namespace --namespace kubefirst --version 2.4.16 --wait kubefirst/kubefirst
```
Comment (johndietz) [inline kubefirst/install.sh:6]:
```suggestion
helm install kubefirst --set=console.installMethod=civo-marketplace --set=global.cloudProvider=civo --set=global.clusterType=bootstrap --set=global.installMethod=civo-marketplace --set=console.ingress.enabled=false  --create-namespace --namespace kubefirst --version 2.4.17 --wait kubefirst/kubefirst
```

File: kubefirst/manifest.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: kubefirst
-version: "v2.4.13"
+version: "v2.4.16"
```
Comment (johndietz) [inline kubefirst/manifest.yaml:3]:
```suggestion
version: "v2.4.17"
```

Comment (johndietz) [inline kubefirst/manifest.yaml:3]:
```suggestion
version: "v2.4.17"
```

**Review by kunal-kushwaha — APPROVED** [review]
lgtm

**Pull request merged**